samples/quote/Makefile

```diff
@@ -9,8 +9,8 @@ SERVER  ?= sampleserver
 
 all: server.json client/sampleclient
 
-server.json: server/Dockerfile signer.pem signer.cer | .acon/
-	$(DOCKER) build -t $(SERVER) -f $< $(PWD)/../..
+server.json: server/Dockerfile $(wildcard server/src/*cpp) signer.pem signer.cer | .acon/
+	$(DOCKER) build -t $(SERVER) -f $< --build-arg https_proxy $(PWD)/../..
```

Comment:
Does it work if only using variable name "--build-arg https_proxy"?

Reply:
Yes, `docker` will pick up the environment variable with the same name.